### PR TITLE
Improving quality and performance by using online tests

### PR DESCRIPTION
**Pull request by cfgnunes**
Online tests used:

https://validator.w3.org/
https://www.dareboost.com/
https://gtmetrix.com/

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -10,16 +10,13 @@
     {% endif %}
 
     <meta charset="utf-8">
-    <meta http-equiv=X-UA-Compatible content="IE=edge,chrome=1">
     <meta name=viewport content="width=device-width, initial-scale=1">
-
-    <meta name=description content="{{ site.bio }}">
```
Comment (sergiokopplin) [inline _layouts/default.html:16]: Why do you removes the description tag?

File: _sass/components/header.sass [hunk 2]
```diff
@@ -7,10 +7,10 @@
 
     > .link > .selfie
         width: 125px
+        height: 125px
         margin-bottom: 25px
         border-radius: 100%
         transition: all .2s
-        box-shadow: 0
```
Comment (sergiokopplin) [inline _sass/components/header.sass:13]: Why that?

File: _sass/pages/tags.sass [hunk 3]
```diff
@@ -26,7 +26,6 @@
         font-weight: 300
         text-transform: uppercase
         text-align: center
-        margin-bottom: 6rem
```
Comment (sergiokopplin) [inline _sass/pages/tags.sass:29]: Why that?

**Review by sergiokopplin — CHANGES_REQUESTED** [review]

File: _sass/components/header.sass (hunk 2 above)
Comment (cfgnunes) [reply _sass/components/header.sass:13]: This tag is redundant. This style modification appears in lines 13 and line 17 of this file.

File: _sass/pages/tags.sass (hunk 3 above)
Comment (cfgnunes) [reply _sass/pages/tags.sass:29]: This is style is override by "margin-bottom: 6rem" in the file "_sass/pages/page.sass". So, this tag is redundant.

File: _layouts/default.html (hunk 1 above)
Comment (cfgnunes) [reply _layouts/default.html:16]: I don't remember exactly. But this modification was made by recommendation of these tests online.

**Pull request merged**